From d0ec77a25f4f20d4d9bb312f07eaf6e98ee1e5be Mon Sep 17 00:00:00 2001
From: kaidoho <kho237115@gmail.com>
Date: Sat, 29 Feb 2020 15:28:09 +0100
Subject: [PATCH] Remove Zephyr from OS list

---
 config.sub | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.sub b/config.sub
index e5573b1a..3b4c7624 100755
--- a/config.sub
+++ b/config.sub
@@ -1365,7 +1365,7 @@ case $os in
 	     | powermax* | dnix* | nx6 | nx7 | sei* | dragonfly* \
 	     | skyos* | haiku* | rdos* | toppers* | drops* | es* \
 	     | onefs* | tirtos* | phoenix* | fuchsia* | redox* | bme* \
-	     | midnightbsd* | amdhsa* | unleashed* | emscripten* | zephyr*)
+	     | midnightbsd* | amdhsa* | unleashed* | emscripten*)
 	# Remember, each alternative MUST END IN *, to match a version number.
 		;;
 	qnx*)
-- 
2.17.1

